### PR TITLE
Model chunking support

### PR DESCRIPTION
**Pull request by billschereriii**

### CODE REVIEW

File: include/client.h [hunk 1]
```diff
@@ -1269,6 +1269,19 @@ class Client : public SRObject
...
 
+        /*!
+        *   \brief Reconfigure the chunking size that Redis uses for model
+        *          serialization, replication, and the model_get command.
+        *   \details This method triggers the AI.CONFIG method in the Redis
+        *             database to change the model chunking size. The default
+        *             size of 511MB should be fine for most applications, so
+        *             it is expected to be very rare that a client calls this
+        *             method.
+        *   \param chunk_size The new chunk size
```
Comment (ashao) [inline include/client.h:1280]: I'm pretty sure the unit is bytes, but let's call that out specifically in the docstring

File: include/pyclient.h [hunk 2]
```diff
@@ -925,6 +925,19 @@ class PyClient : public PySRObject
...
 
+        /*!
+        *   \brief Reconfigure the chunking size that Redis uses for model
+        *          serialization, replication, and the model_get command.
+        *   \details This method triggers the AI.CONFIG method in the Redis
+        *             database to change the model chunking size. The default
+        *             size of 511MB should be fine for most applications, so
+        *             it is expected to be very rare that a client calls this
+        *             method.
+        *   \param chunk_size The new chunk size
```
Comment (ashao) [inline include/pyclient.h:936]: Same here

File: include/redisserver.h [hunk 3]
```diff
@@ -520,6 +520,33 @@ class RedisServer {
...
+
+        /*!
+        *   \brief Reconfigure the chunking size that Redis uses for model
+        *          serialization, replication, and the model_get command.
+        *   \details This method triggers the AI.CONFIG method in the Redis
+        *             database to change the model chunking size. The default
+        *             size of 511MB should be fine for most applications, so
+        *             it is expected to be very rare that a client calls this
+        *             method.
+        *   \param chunk_size The new chunk size
```
Comment (ashao) [inline include/redisserver.h:537]: Units here

**Review by ashao — APPROVED** [review]
Minor comment about adding in the units. Also as we agree offline, in the documentation please mention that setting the chunksize is generally not needed, even by advanced users; instead highlight that now we support users uploading models >=512MB.

File: include/client.h (hunk 1 above)
Comment (billschereriii) [reply include/client.h:1280]: Addressed in all 6 places where needed

File: include/pyclient.h (hunk 2 above)
Comment (billschereriii) [reply include/pyclient.h:936]: Addressed in all 6 places where needed

File: include/redisserver.h (hunk 3 above)
Comment (billschereriii) [reply include/redisserver.h:537]: Addressed in all 6 places where needed

**Pull request merged**